src/client/pages/EmailSent.tsx

```diff
@@ -1,99 +1,104 @@
-  const [loading, setLoading] = useState<boolean>(false);
 
   useEffect(() => {
     setHasJS(true);
   }, []);
 
-  const onResendClick = () => {
-    setLoading(true);
-  };
-
   return (
-    <>
-      <Header />
-      <Main
-        subTitle={subTitle}
-        successOverride={loading ? undefined : 'Email sent'}
-      >
-        <PageBox>
-          <PageHeader>Check your email inbox</PageHeader>
-          <PageBody>
-            {email ? (
-              <PageBodyText>We’ve sent an email to {email}.</PageBodyText>
-            ) : (
-              <PageBodyText>We’ve sent you an email.</PageBodyText>
-            )}
-            <PageBodyText>
-              Please follow the instructions in this email. If you can&apos;t
-              find it, it may be in your spam folder.
-            </PageBodyText>
-            <PageBodyText>The link is only valid for 30 minutes.</PageBodyText>
-            {email && resendEmailAction && hasJS && (
-              <form method="post" action={`${resendEmailAction}${queryString}`}>
-                <CsrfFormField />
-                <TextInput
-                  label=""
-                  name="email"
-                  type="email"
-                  value={email}
-                  hidden={true}
-                />
-                <br />
-                <br />
-                <Button
-                  css={button}
-                  type="submit"
-                  onClick={onResendClick}
-                  data-cy="resend-email-button"
-                >
-                  {loading ? 'Resending...' : 'Resend email'}
-                </Button>
-                <br />
-                <br />
-              </form>
-            )}
-            {previousPage && (
-              <PageBodyText>
-                Wrong email address?{' '}
-                <Link subdued={true} href={previousPage}>
-                  Change email address
-                </Link>
-              </PageBodyText>
-            )}
-          </PageBody>
-        </PageBox>
-      </Main>
-      <Footer />
-    </>
+    <MainLayout
+      pageTitle="Check your email inbox"
+      successOverride={showSuccess ? 'Email sent' : undefined}
+      errorOverride={errorMessage}
+    >
+      {email ? (
+        <MainBodyText>
+          We’ve sent an email to <b>{email}</b>.
+        </MainBodyText>
+      ) : (
+        <MainBodyText>We’ve sent you an email.</MainBodyText>
+      )}
+      <MainBodyText>
+        Please follow the instructions in this email. If you can’t find it, it
+        may be in your spam folder.
+      </MainBodyText>
+      <MainBodyText>
+        <b>This link is only valid for 30 minutes.</b>
+      </MainBodyText>
+      {previousPage && (
+        <MainBodyText>
+          Wrong email address?{' '}
+          <Link subdued href={previousPage}>
+            Change email address
+          </Link>
+        </MainBodyText>
+      )}
+      {email && resendEmailAction && hasJS && (
```

Comment:
just wondering if we still need to check for client side JS with `hasJS` on this one now that the `{loading ? 'Resending...' : 'Resend email'}` bit is gone?

Reply:
you're right! it's not needed anymore, so I'll remove it.